src/ethereum/api/thirdParty.ts

```diff
@@ -71,6 +82,18 @@ export class ThirdPartyAPI extends BaseGraphAPI {
     })
     return thirdParties.length > 0
   }
+
+  fetchTiers = async (): Promise<TierFragment[]> => {
+    const {
+      data: { tiers = [] },
+    } = await this.query<{
```

Comment:
Should this be paginated? It's "free" using `this.paginate()`
I'm not sure how many tiers are there going to be (the `thegraph` max is 1k)

Reply:
Done. I've paginated the query and changed the path to `/v1/tiers/thirdParty`